doc/source/tune/_tutorials/tune-serve-integration-mnist.py

```diff
@@ -615,7 +615,7 @@ def get_current_model(model_dir):
     args = parser.parse_args()
 
     if args.smoke_test:
-        ray.init(num_cpus=2)
+        ray.init(num_cpus=3)
```

Comment:
cc @simon-mo does the http proxy (https://github.com/ray-project/ray/commit/39813ff6b0c960d9c3f86c0c7ecb4ff63d5e6887) occupy a CPU, and is this expected?